server/src/main/java/org/elasticsearch/cluster/action/shard/ShardStateAction.java

```diff
@@ -379,7 +379,7 @@ public void clusterStatePublished(ClusterChangedEvent clusterChangedEvent) {
                 if (logger.isTraceEnabled()) {
                     logger.trace("{}, scheduling a reroute", reason);
                 }
-                routingService.reroute(reason);
+                routingService.reroute(reason, ActionListener.wrap(() -> logger.trace("{}, reroute completed", reason)));
```

Comment:
this also logs the same line on an exception :/
I would prefer two different log lines, and the failure one with the exception (same for other places in this PR)

Reply:
![image](https://user-images.githubusercontent.com/5058284/60193659-facaf280-982f-11e9-8ecb-6a3c91652d57.png)

Fixed in ce5946b